### ISSUE TITLE
Loss Function

### ISSUE THREAD

**Issue by sghong977** [sghong977/AIML_Basics#4]
### 4. loss function
- [cost function](https://ynebula.tistory.com/28)

**regression**이면 MSE, MAE 등.   (이외에도 regularization term에 비슷한 값 나오라고 L2, L1 norm 넣거나)  

**classification**이면 주로 Cross Entropy Loss.  

Cosine Similarity도 많이 쓴다.  
self-supervised learning할때 augmentation 다양하게 적용하고 feature들 간에 비슷하라고.  

### 생각해볼 것
- Supervised learning과 Unsupervised learning의 Objective fuction과 그 차이를 딥러닝 모델 중심으로 설명해 주세요
